lib/shortcodes/property-attribute.php

```diff
@@ -157,6 +164,11 @@ public function call( $atts = "" ) {
           $return[ 'before' ] = html_entity_decode( $args[ 'before' ] );
         }
 
+        // show label if  'show_taxonomy_label' true
+        if($args['show_taxonomy_label'] && $args['show_taxonomy_label'] !== "false" ) {
```

Comment:
Although in this case it may not matter but I like to be over-paranoid and wrap such statements into ```isset( $args['show_taxonomy_label'] )```